### PR TITLE
Migration from blockcypher to insight [ch1256]

### PR DESCRIPTION
**Pull request by trojkat**

### CODE REVIEW

File: clove/network/bitcoin_based/ravencoin.py [hunk 1]
```diff
@@ -23,12 +22,8 @@ class Ravencoin(InsightAPIv4, BitcoinBaseNetwork):
         'SECRET_KEY': 128
     }
     source_code_url = 'https://github.com/RavenProject/Ravencoin/blob/master/src/chainparams.cpp'
-    api_url = 'https://ravencoin.network'
-    api_prefix = '/api'
-
-    @classmethod
-    def get_fee(cls):
-        return clove_req_json(f'{cls.api_url}{cls.api_prefix}/utils/estimatesmartfee?nbBlocks=1')['1']
```
Comment (pafff) [inline clove/network/bitcoin_based/ravencoin.py:31]: Why is this removed?

Comment (trojkat) [reply clove/network/bitcoin_based/ravencoin.py]: This method was here to support the case when there was a different prefix for API and UI. I created two variables and now we have `api_url`, `ui_url` and `InsightAPIv4` is doing the rest.

Comment (trojkat) [reply clove/network/bitcoin_based/ravencoin.py:31]: You are right, Ravencoin has a different endpoint for fee. I'll revert this change.

**Review by jaworows — APPROVED** [review]

**Review by pafff — APPROVED** [review]
